pyproject.toml

```diff
@@ -3,7 +3,7 @@ line-length = 119
 
 [tool.poetry]
 name = "dbt-copilot-tools"
-version = "0.1.15"
+version = "0.1.16"
```

Comment:
You will need to rebase `main` as you're a few versions behind!

Reply:
Yep, I'll do that once the previous PR is reviewed & merged.